plugins/Makefile

```diff
@@ -5,3 +5,8 @@ test:
 .PHONY: install-all-dev
 install-all-dev:
 	pip install -e .
+
+.PHONY: build_all_plugins
+build_all_plugins:
```

Comment:
how about run tests for all plugins?

Reply:
can i do it separately?  it's currently run by the top level Makefile.